src/framework/musesampler/internal/musesamplerwrapper.cpp

```diff
@@ -256,12 +256,9 @@ void MuseSamplerWrapper::setIsActive(bool arg)
 
     m_samplerLib->setPlaying(m_sampler, arg);
 
-    if (!isActive()) {
-        //! NOTE: restore the current position because setPlaying(m_sampler, false) resets it
+    if (arg) {
```

Comment:
Very minor point - I'd personally prefer a rename from `arg` to `active`. To me it's slightly more readable.